internal/planner/configuration.go

```diff
@@ -147,6 +147,24 @@ func (pl *Planner) Update() (changed bool, err error) {
 	return
 }
 
+// Prune the target share from the configuration.
+func (pl *Planner) Prune() (changed bool, err error) {
```

Comment:
I find it a bit confusing that the function returns `err error` but it is not defined anywhere within the function's body. But maybe it is just me still needs to adjust my eyes to Go style.

Reply:
I often don't use that style much, but every once in a while I think it's clearer.  In this case I'm not married to it.